VSRAD.Package/Commands/BreakpointMenuCommand.cs

```diff
@@ -0,0 +1,78 @@
+﻿using EnvDTE;
+using Microsoft;
+using Microsoft.VisualStudio.OLE.Interop;
+using Microsoft.VisualStudio.ProjectSystem;
+using Microsoft.VisualStudio.Shell;
+using System;
+using System.ComponentModel.Composition;
+using VSRAD.Package.Utils;
+using VSRAD.Package.ProjectSystem;
+
+namespace VSRAD.Package.Commands
+{
+    [Export(typeof(ICommandHandler))]
+    [AppliesTo(Constants.RadOrVisualCProjectCapability)]
+    public sealed class BreakpointMenuCommand : ICommandHandler
+    {
+        private readonly SVsServiceProvider _serviceProvider;
+        private readonly IActiveCodeEditor _codeEditor;
+        private readonly IBreakpointTracker _breakpointTracker;
+
+        [ImportingConstructor]
+        public BreakpointMenuCommand(SVsServiceProvider serviceProvider, IActiveCodeEditor codeEditor, IBreakpointTracker breakpointTracker)
+        {
+            _serviceProvider = serviceProvider;
+            _codeEditor = codeEditor;
+            _breakpointTracker = breakpointTracker;
+
+        }
+
+        public Guid CommandSet => Constants.BreakpointMenuCommandSet;
+
+        public OLECMDF GetCommandStatus(uint commandId, IntPtr commandText)
+        {
+            ThreadHelper.ThrowIfNotOnUIThread();
+            var dte = _serviceProvider.GetService(typeof(DTE)) as DTE;
+            Assumes.Present(dte);
+
+            var flags = OleCommandText.GetFlags(commandText);
+
+            if (commandId == Constants.BreakpointMenuToggleResumable)
+            {
+                var line = _codeEditor.GetCurrentLine() + 1;
```

Comment:
I strongly recommend storing the breakpoints with 0-based lines, and only add 1 when interfacing with DTE. This may seem arbitrary, but all proper VS APIs like `IVsTextView` use 0-based indexing, and we follow that convention in our code as well, including `BreakpointTracker`. It's only DTE that's an outlier with 1-based indexing for everything.

Reply:
It's just that the first numbering I encountered was in DTE. 
Fixed, thanks!